content/providers/hardware.json

```diff
@@ -0,0 +1,162 @@
+[
+  {
+    "title": "IBM Qiskit Runtime",
+    "image": "static/images/providers/qiskit-ibm-runtime.png",
+    "description": "Qiskit Runtime is a quantum computing service and programming model that allows users to optimize workloads and efficiently execute them on quantum systems at scale. The programming model extends the existing interface in Qiskit with a set of new primitive programs.",
+    "installation": "pip install qiskit-ibm-runtime",
+    "websiteCta": {
+      "label": "Website",
+      "url": "https://quantum-computing.ibm.com/lab/docs/iql/runtime/"
+    },
+    "docCta": {
+      "label": "Docs",
+      "url": "https://qiskit.org/documentation/partners/qiskit_ibm_runtime/"
+    },
+    "sourceCta": {
+      "label": "GitHub",
+      "url": "https://github.com/Qiskit/qiskit-ibm-runtime"
+    },
+    "helloWorldExample": [
+      "from qiskit import QuantumCircuit\n",
+      "from qiskit_ibm_runtime import QiskitRuntimeService, Sampler\n",
```

Comment:
I think you don't need to add `\n` at the end of the line, we can manage it on the code. @techtolentino , I know you fixed one issue to have the commands on different lines, could you confirm what I'm saying?

Reply:
Sure, sounds good!

Reply:
@y4izus Instead of an array of strings, is it ok to put only one string? I am planning to write the entries as yaml (because it's easier to write, no need to worry about curly brackets and quotes)and convert to json locally before pushing to this repo.

This is the most straightward output as one lone string with escape characters like `\n`

yaml:

```yaml
helloWorldExample: >-
  from qiskit import QuantumCircuit
  from qiskit_ibm_runtime import QiskitRuntimeService, Sampler

  # Save an IBM Quantum account
  QiskitRuntimeService.save_account(channel="ibm_quantum", token="MY_IBM_QUANTUM_TOKEN")

  # Select a backend
  service = QiskitRuntimeService()
  backend = service.backend("ibmq_qasm_simulator")

  # Build a circuit
  circuit = QuantumCircuit(2, 2)
  circuit.h(0)
  circuit.cx(0,1)
  circuit.measure([0,1], [0,1])

  # Run the circuit and get result distribution
  sampler = Sampler(session=backend)
  job = sampler.run(circuit)
  quasi_dist = job.result().quasi_dists[0]
  print(quasi_dist)
```

```json
"helloWorldExample": "from qiskit import QuantumCircuit from qiskit_ibm_runtime import QiskitRuntimeService, Sampler\n# Save an IBM Quantum account QiskitRuntimeService.save_account(channel=\"ibm_quantum\", token=\"MY_IBM_QUANTUM_TOKEN\")\n# Select a backend service = QiskitRuntimeService() backend = service.backend(\"ibmq_qasm_simulator\")\n# Build a circuit circuit = QuantumCircuit(2, 2) circuit.h(0) circuit.cx(0,1) circuit.measure([0,1], [0,1])\n# Run the circuit and get result distribution sampler = Sampler(session=backend) job = sampler.run(circuit) quasi_dist = job.result().quasi_dists[0] print(quasi_dist)"
```

Reply:
After discussion, we settled with array of strings. I have figured out a way export to array of strings in json.

Reply:
Fixed in https://github.com/Qiskit/qiskit.org/pull/2968/commits/60fb2c5174ff6e20fb77418e13bb75ca4fa0ea89